test/services/permissions.test.js

```diff
@@ -1,6 +1,9 @@
 const nock = require('nock');
 const PermissionsChecker = require('../../src/services/permissions-checker');
 const context = require('../../src/context/');
+const initContext = require('../../src/context/init');
+
+context.init(initContext);
```

Comment:
This init (and the two below at the top level describe) should be done in a before hook.

Reply:
I'm totally agree with you, but the hooks is not allowed in `forest-express`

Reply:
I can move this in the describe function instead ?